### ISSUE TITLE
Have "spack spec" indicate which specs are not yet built

### ISSUE THREAD

**Issue by citibeth** [LLNL/spack#483]
For example... instead of:

```
  py-scipy@0.15.1%gcc@4.9.3=linux-x86_64
      ^netlib-blas@3.5.0%gcc@4.9.3+fpic=linux-x86_64
      ^netlib-lapack@3.5.0%gcc@4.9.3+shared=linux-x86_64
          ^cmake@3.4.3%gcc@4.9.3+ncurses=linux-x86_64
              ^ncurses@6.0%gcc@4.9.3=linux-x86_64
      ^py-nose@1.3.6%gcc@4.9.3=linux-x86_64
          ^py-setuptools@19.2%gcc@4.9.3=linux-x86_64
              ^python@3.5.1%gcc@4.9.3=linux-x86_64
                  ^bzip2@1.0.6%gcc@4.9.3=linux-x86_64
                  ^readline@6.3%gcc@4.9.3=linux-x86_64
                  ^sqlite@3.8.5%gcc@4.9.3=linux-x86_64
                  ^zlib@1.2.8%gcc@4.9.3=linux-x86_64
      ^py-numpy@1.9.2%gcc@4.9.3+blas=linux-x86_64
```

We should get something like:

```
*  py-scipy@0.15.1%gcc@4.9.3=linux-x86_64
      ^netlib-blas@3.5.0%gcc@4.9.3+fpic=linux-x86_64
      ^netlib-lapack@3.5.0%gcc@4.9.3+shared=linux-x86_64
          ^cmake@3.4.3%gcc@4.9.3+ncurses=linux-x86_64
              ^ncurses@6.0%gcc@4.9.3=linux-x86_64
      ^py-nose@1.3.6%gcc@4.9.3=linux-x86_64
          ^py-setuptools@19.2%gcc@4.9.3=linux-x86_64
              ^python@3.5.1%gcc@4.9.3=linux-x86_64
                  ^bzip2@1.0.6%gcc@4.9.3=linux-x86_64
                  ^readline@6.3%gcc@4.9.3=linux-x86_64
                  ^sqlite@3.8.5%gcc@4.9.3=linux-x86_64
                  ^zlib@1.2.8%gcc@4.9.3=linux-x86_64
      ^py-numpy@1.9.2%gcc@4.9.3+blas=linux-x86_64
```

This would make it easier to see if a spec is right (i.e. consistent with other software we want installed).  PR #120 will ameliorate this issue, but it still seems like an nice feature.

**Comment by tgamblin** [comment]
:+1: This and more detailed output from `spack install` (e.g., showing what will be built in advance) would be good.

**Comment by adamjstewart** [comment]
Closed by #2238. This feature is now available with `spack spec -I`
